code/hud/hudtargetbox.cpp

```diff
@@ -569,10 +569,6 @@ void HudGaugeTargetBox::renderTargetShip(object *target_objp)
 		render_info.set_object_number(OBJ_INDEX(target_objp));
 
 		switch (Targetbox_wire) {
-			case 0:
```

Comment:
Why not simply make `Targetbox_wire` default to 2, rather than getting rid of case 0?